src/resmom_win/start_exec.c

```diff
@@ -3151,10 +3089,12 @@ start_process(task *ptask, char **argv, char **envp, bool nodemux)
 		si.hStdOutput = (HANDLE)_get_osfhandle(script_out);
 		si.hStdError = (HANDLE)_get_osfhandle(script_err);
 		/* set stdout and stderr to append mode */
-		SetFilePointer(si.hStdOutput, (LONG)NULL,
-			(PLONG)NULL, FILE_END);
-		SetFilePointer(si.hStdError, (LONG)NULL,
-			(PLONG)NULL, FILE_END);
+		if (SetFilePointer(si.hStdOutput, (LONG)NULL,
+			(PLONG)NULL, FILE_END) == INVALID_SET_FILE_POINTER)
+			log_err(-1, __func__, "Unable to SetFilePointer");
+		if (SetFilePointer(si.hStdError, (LONG)NULL,
+			(PLONG)NULL, FILE_END) == INVALID_SET_FILE_POINTER)
+			log_err(-1, __func__, "Unable to SetFilePointer");
 	}
 
 	/* If we fail to get cmd shell(unlikely), use "cmd.exe" as shell */
```

Comment:
line 3266, 3276 where rc = CreateProcessAsUser. if it fails error logging is not required?. If missed then check in other places in this file.

Reply:
All the CreateProcess & CreateProcessAsUser failures are logged together at 3299